src/objects/Mesh.js

```diff
@@ -72,6 +72,16 @@ class Mesh extends Object3D {
 
 	}
 
+	dispose() {
+
+		this.geometry.dispose();
+
+		this.material.dispose();
```

Comment:
Similar issue with textures. Materials and geometries can be shared with other meshes so it is not correct to call `dispose()` on these object here.